express-notes-api/express-notes-api/index.js

```diff
@@ -0,0 +1,93 @@
+const express = require('express');
+const fs = require('fs');
+const data = require('./data.json');
+const app = express();
+
+app.get('/api/notes', (req, res) => {
+  const jsonArray = [];
+  for (const key in data) {
+    jsonArray.push(data[key]);
+  }
+  res.json(jsonArray);
+  res.status(200);
```

Comment:
You want to set the status first, then send the response, this won't actually change the status when you do it after data is sent. In this case you don't need to set the status code because 200 is the default status code, so you can just remove line 12 here.